### ISSUE TITLE
Strong walls in single player mode plus more ox tethers per quarry for AI

### ISSUE THREAD

**Issue by bakpx9090** [Sh0wdown/UnofficialCrusaderPatch#622]
First off, thank you so much to the team for the UCP ! 
I am thinking, is it possible to have an option for strong walls (either by reducing the melee damage of units or turning it off altogether like in multiplayer ?) 
Also AI only builds one tether per quarry, is there anyway to increase this to 3 or 4 per quarry ?

**Comment by Heroesflorian** [comment]
It would probably be possible, but until the AI is taught to use their siege equipment more competently, I am not sure how useful the option would actually be. 
Just imagine the Rat or Sultan attacking without any siege engines when the castle walls are resistant or even immune to melee damage. And even stronger AIs would struggle to bring down castles with strong walls. 

With gates and possibly towers unaffected by this option, it might work, but still it's probably not as simple as just turning on strong walls and everything working well without further adjustments. Anyway, yeah, would love to play around with it... 

Additional ox tethers are a good point as well; however, afaik the AI does actually not just build one ox tether per quarry.

**Comment by bakpx9090** [comment]
Yeah i had this idea actually after using the AIV editor and forcing the weaker lords to actually use more laddermen (phillip, pig, sheriff) - it was actually quite fun to see macemen swamps the castle and try different ways of sieging. Currently however the laddermen are not as useful since you can easily use other melee unit to simply break the wall and get through, rather than climbing it (the fact that they can break the walls always felt so cheap to me).

On the quarries, i m pretty sure all the AIs i have modded only build the right number of quarries that i have set, but only one tether per quarry, leading to an inefficient economy. Given that quarries often yield the best returns if this can be fixed the AIs will have a way more balanced economies late game !
